src/components/common/TextArea/TextAreaStyle.tsx

```diff
@@ -0,0 +1,40 @@
+import { STYLE_CONSTANTS } from '@constants/styleConstants';
+import styled from 'styled-components';
+
+export const TextAreaWrapper = styled.div`
+  width: 100%;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  textarea {
```

Comment:
why styles are here but not in the textarea itself?

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/951221ce3504af9e7a903b4a7c165220536e955f#diff-526d33de93204a6fc68e39d8ecbdf82532a7206dcd9c3c1648b3a00a5d6b35f8